lms/djangoapps/certificates/views/webview.py

```diff
@@ -669,6 +679,37 @@ def render_html_view(request, course_id, certificate=None):  # pylint: disable=t
         return response
 
 
+def _update_certificate_configuration_from_credentials(
+    configuration: Dict[str, Union[str, List[Dict[str, str]]]],
+    course: 'CourseBlockWithMixins', mode: str
+) -> Dict[str, Union[str, List[Dict[str, str]]]]:
+    """
+    Populates certificate configuration from credentials service.
+
+    If config not found in credentials returns configuration without changes.
+    """
+    try:
+        response = get_certificate_configuration_from_credentials(course.id, mode)
+        response.raise_for_status()
+    except Exception as err:   # pylint: disable=broad-except
+        log.warning(f'Failed to get course certificate config for course {course.id} from Credentials. {err}')
```

Comment:
But you already have the log
```
log.warning(
            'There is no certificate configuration in credentials for a course [%s] with a mod [%s]',
```
Why do we need this log?
Maybe in that function, the log is not needed and there is no need to intercept anything. Let it throw an exception. But in this place, you will intercept, if that?

Reply:
Removed after using get_api_data.